### ISSUE TITLE
[Google Cloud Script] myvalue.yaml and values.yaml

### ISSUE THREAD

**Issue by LucianaMarques** [mlbench/mlbench-helm#5]
Hi there, 

Just a docs suggestion. I found a little confusing the 'values.yaml' file in the repo when we are supposed to create a 'myfiles.yaml', which is used in the google cloud script. Is there any specific reason for this to be so? Because if not, I think it's simpler if we leave a myvalues.yaml unique file and the user can edit it with his or her preferences. 

I think it's better if the script does the whole work for user in setup, that is, don't needing to do anything extra apart from running the script.

**Comment by Panaetius** [comment]
It's just what Helm does, see https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing though they call it `config.yaml`. So that's the official way to do it. `values.yaml` is for default settings and to list available settings, `myvalues.yaml` is for custom settings and settings where there is no sensible default value. We could add an `myvalues.yaml` that just contains all the mandatory values, but empty. Adding default values would be besides the point, since by definition it's for values that have no sensible default, like number of workers (We don't know the layout of users' clusters).

I can see how that would help, so I think we should add it with blank entries. And let's call it `config.yaml` though to keep in line with official docs.

Since custom values can also be set from the shell, having the scripts not depend on `myvalues.yaml` is a great suggestion and can likely be done alongside the `config.yaml`, i.e. the script still uses the provided `config.yaml` AND have parameters to the script that can override it, too.

The reasoning behind this is that Helm charts follow a specific layout and hopefully our chart will be a part of the official Helm repository at some point, in which case users would use Helm to install MLBench, which precludes the use of the scripts. So the Chart itself has to follow Helm conventions and be compatible with them. The scripts are separate from this and provided as a convenience, but they can't be an integral part of the Helm part of the application itself, so we can't get rid of the Helm specific parts in order to fit the scripts.

As for why Helm, it's [by far the most popular solution for deployment of K8s applications](https://trends.google.com/trends/explore?geo=US&q=helm%20kubernetes,draft%20kubernetes,ksonnet,metaparticle,Skaffold)

So basically, yes, we should do both of your suggestions, we just have to pay a little attention on how it's done.

**Comment by Panaetius** [comment]
I've committed a `config.yaml` file that only contains the two most necessary values that a user needs to set and the `google_cloud_setup.sh` script now sets those values based on the parameters supplied to the script, so the user should not have to change the `config.yaml`, except for overriding more advanced settings/features.

Let me know what you think.

**Comment by martinjaggi** [comment]
thanks! sounds great

**Comment by Panaetius** [comment]
Setup script now uses supplied parameters to override values specified in `config.yaml` plus `config.yaml`is now part of the repository, so it's not missing when the repo gets pulled.
